Jenkinsfile

```diff
@@ -407,7 +407,22 @@ try {
           }
         }
       },
-      'Python 2: CPU Win':{
+      'Julia 0.6: CPU': {
```

Comment:
https://builds.apache.org/blue/organizations/jenkins/incubator-mxnet/detail/PR-8175/5/pipeline

The graph of pipeline does not include Julia. Do I miss something?

Reply:
Figured out this: https://support.cloudbees.com/hc/en-us/articles/227246367-Why-Jenkinsfile-changes-Are-Not-Reflected-in-PR-Build
I do not have permission to do that...